src/java/org/apache/cassandra/metrics/ColumnFamilyMetrics.java

```diff
@@ -737,6 +743,27 @@ public Double getValue()
                 return cfs.getTombstoneRatio();
             }
         });
+        droppableTombstoneCount = createColumnFamilyGauge("DroppableTombstoneCount", new Gauge<Double>()
+        {
+            public Double getValue()
+            {
+                return cfs.getDroppableTombstoneCount();
+            }
+        });
+        liveTombstoneCount = createColumnFamilyGauge("LiveTombstoneCount", new Gauge<Double>()
+        {
+            public Double getValue()
+            {
+                return cfs.getLiveTombstoneCount();
+            }
+        });
+        tombstoneCount = createColumnFamilyGauge("TombstoneCount", new Gauge<Double>()
+        {
+            public Double getValue()
+            {
+                return cfs.getTombstoneCount();
+            }
+        });
```

Comment:
these can all be method refs

Reply:
ah i wanted to stay consistent with the current style. i think it'd look weird to have `tombstoneCount = createColumnFamilyGauge("TombstoneCount", cfs::tombstoneCount);` here

Reply:
intellij has a thing where you can just go through and convert the existing in the method for consistency; there's other method refs in the codebase so i think this is just a holdover

Reply:
converted just the 6 metrics that we added